resolvers/uploadYear.js

```diff
@@ -490,73 +162,78 @@ const uploadYear = async (_, { year, projects, data }) => {
     }
   `);
 
+  // Log any errors
+  if (metricData.errors) {
```

Comment:
I added this condition here to help us debugging in the future. It will catch any errors and display them on the front end for us.